config/kraft/broker.properties

```diff
@@ -31,22 +31,23 @@ controller.quorum.voters=1@localhost:9093
 
 ############################# Socket Server Settings #############################
 
-# The address the socket server listens on. It will get the value returned from
-# java.net.InetAddress.getCanonicalHostName() if not configured.
+# The address the socket server listens on. If not configured, the host name will get from
+# java.net.InetAddress.getCanonicalHostName(), with PLAINTEXT listener name, and port 9092.
```

Comment:
Make it clear about which default value will be used if not set.

Reply:
s/the host name will get from/the host name will be equal to the value of/